cmd/ipfs/daemon.go

```diff
@@ -104,23 +104,18 @@ func daemonFunc(req cmds.Request, res cmds.Response) {
 		}
 	}
 
-	// To ensure that IPFS has been initialized, fetch the config. Do this
-	// _before_ acquiring the daemon lock so the user gets an appropriate error
-	// message.
-	// NB: It's safe to read the config without the daemon lock, but not safe
-	// to write.
```

Comment:
the errors returned may now differ, but they actually mean something now. the other error was completely worthless